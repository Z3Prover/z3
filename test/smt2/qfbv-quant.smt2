;; Variation of GitHub Issue #544; Nuno wants quantifiers in qfbv
(assert (forall ((X (_ BitVec 32)))
            (exists ((Y (_ BitVec 32)))
                (= X (bvmul Y Y)))))
(apply qfbv)
