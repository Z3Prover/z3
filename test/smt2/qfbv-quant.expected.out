(goals
(goal
  false
  :precision precise :depth 7)
)
